### PR TITLE
Correct text alignment in RTL languages

### PR DESCRIPTION
**Pull request by luixxiul**
For https://github.com/vector-im/element-web/issues/14520 and https://github.com/vector-im/element-web/issues/4771

This PR intends to relieve frustration of RTL language users.
cf. https://github.com/matrix-org/matrix-react-sdk/pull/5453

Please mind the position of `(edited)`.
cf. https://github.com/matrix-org/matrix-react-sdk/pull/5453#issuecomment-862466809

I modified the declarations for `blockquote`, `ul`, and `ol` to apply logical properties. Since I am not able to read RTL languages, it is not really possible for me to test edge cases though.

The ideal solution would be to implement logical properties all over the UI, but the task is too large for this PR. Therefore adjusting position of UI elements such as timestamp is out of scope.

| |Before|After|
|-|---------|------|
|IRC layout|before1|Untitled2|
|Modern layout|before2|Untitled|
|Bubble layout|before3|Untitled3|

type: defect

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Correct text alignment in RTL languages ([\#9026](https://github.com/matrix-org/matrix-react-sdk/pull/9026)). Contributed by @luixxiul.

### CODE REVIEW

**Comment by luixxiul** [comment]
Though I don't find a regression, I think it would be better to merge this PR after some feedback comments are provided from native RTL language speakers.

File: res/css/views/rooms/_EventTile.scss [hunk 1]
```diff
@@ -485,6 +490,11 @@ $left-gutter: 64px;
         color: inherit; // inherit the colour from the dark or light theme by default (but not for code blocks)
         font-size: $font-14px;
 
+        ol,
+        ul {
```
Comment (turt2live) [inline res/css/views/rooms/_EventTile.scss:494]: we should add comments for what this is targeting - it's not immediately obvious what the `ol` and `ul` actually are.

**Review by turt2live — COMMENTED** [review]
we've had massive issues with this sort of change in the past - I'd very much like to see screenshot tests of the timeline using various sample messages to ensure we didn't break anything.

[#element-dev:matrix.org](https://matrix.to/#/#element-dev:matrix.org) will likely be best to talk more about those sorts of tests.

**Comment by luixxiul** [comment]
Do we have an example of that kind of tests? Though I am not able to create one from nothing, I could at least try to modify the example.

**Comment by luixxiul** [comment]
converting to draft for now- looking for a way of implementing visual tests.

**Comment by turt2live** [comment]
after the weekend the team should be able to help with tests in https://matrix.to/#/#element-dev:matrix.org

**Comment by luixxiul** [comment]
Closed as RTL design will never land at least until Element X
